moveos/moveos/src/moveos.rs

```diff
@@ -371,12 +384,19 @@ impl MoveOS {
         let mut session = self.vm.new_session(&self.db);
         //TODO limit the view function max gas usage
         let mut gas_meter = UnmeteredGasMeter;
-        let result = session.execute_function_bypass_visibility(
-            module,
-            function_name,
+        //View function use a fix address and fix hash
+        let tx_context = TxContext::new(AccountAddress::ZERO, H256::zero());
```

Comment:
using the default mock TxContext will have problems in some scenarios, such as account_storage::exist_account_storage depends on the real user address.

Therefore, the view function needs to pass the real address as a parameter.

```
    /// check if account storage eixst
    public fun exist_account_storage(ctx: &mut StorageContext, account: address): bool {
        let object_id = object::address_to_object_id(account);
        let object_storage = storage_context::object_storage_mut(ctx);
        object_storage::contains<AccountStorage>(object_storage, object_id)
    }
```